gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFileSystemConfiguration.java

```diff
@@ -292,6 +295,12 @@
   public static final HadoopConfigurationProperty<Boolean> GCS_OUTPUT_STREAM_DIRECT_UPLOAD_ENABLE =
       new HadoopConfigurationProperty<>("fs.gs.outputstream.direct.upload.enable", false);
 
+  /** Configuration key for the minimal time interval between consecutive sync/hsync/hflush
```

Comment:
Formate code with Google Java Format: https://github.com/google/google-java-format

To format only changed line you need to use command:
```
git diff -U0 upstream/master | python ~/dev/src/google-java-format/scripts/google-java-format-diff.py -p1 -i
```

Reply:
done.